EnglishFilter/sections/adservers.txt

```diff
@@ -5,6 +5,7 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||searsbushelbus.com^$third-party
```

Comment:
Please move the rule to japanese adservers

Reply:
Fixed.